src/Sulu/Bundle/AdminBundle/Resources/js/utils/Ajv/formats/idnEmailValidator.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import IsEmail from 'isemail';
+import {FormatValidator} from 'ajv';
+
+const idnEmailValidator: FormatValidator = (data: string): boolean => {
```

Comment:
would prefer to update `ajv` and add the `ajv-formats` and `ajv-formats-draft2019` package. just had a look at the changelog, i think the update should not cause problems 🙂

Reply:
I would prefer not to install `ajv-formats-draft2019`, because it contains formats which are not working as expected, see https://github.com/sulu/sulu/issues/5681#issuecomment-777442172 ... So we still need to implement the `idn-email` format by ourselves, but it's not that complicated with using `isemail`, so that should be fine

Reply:
just saw that the `ajv-formats-draft2019` is a community package. is okay for me too 🙂